html/elements/datalist.json

```diff
@@ -18,7 +18,8 @@
               "version_added": "4"
             },
             "firefox_android": {
-              "version_added": "4"
+              "version_added": "4",
+              "notes": "The dropdown menu containing available options does not appear in Firefox for Android starting version 67 - see <a href='https://bugzil.la/1535985'>bug 1535985</a>."
```

Comment:
```suggestion
              "notes": "Since Firefox for Android 79, the dropdown menu containing available options does not appear. See <a href='https://bugzil.la/1535985'>bug 1535985</a>."
```

Reply:
I have no way to test all the way back to those versions and instead, relied on the bug report to state that the problem started in v67. However, if you have tested it and are confident, that's great and I'm happy to commit this. In a second! 😊